### ISSUE TITLE
Entry Plain keyboard still autocorrects when using Microsoft SwiftKey

### ISSUE THREAD

**Issue by kklose23** [dotnet/maui#19696]
### Description

An entry with a plain keyboard is still autocorrecting when I use Microsoft SwiftKey on my Android.  Is this something that needs to be fixed on the MAUI side of things?

### Version with bug

8.0.3

### Is this a regression from previous behavior?

Not sure, did not test other versions

### Last version that worked well

Unknown/Other

### Affected platforms

Android

### Did you find any workaround?

No

**Comment by mattleibow** [comment]
Adding the try latest version because we did so many entry fixes this year.

Also, does this happen with the default OS keyboard? If not, then the MS keyboard may not be respecting the OS requirements, so maybe it was fixed in an app update?

**Comment by kklose23** [comment]
@mattleibow Just tested with the .NET 9 version of MAUI, and unfortunately, it's still autocorrecting when using a Plain keyboard on an Entry

**Comment by mattleibow** [comment]
Did you set `IsTextPredictionEnabled ` to false?

https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/entry?view=net-maui-9.0#enable-and-disable-text-prediction

**Comment by kklose23** [comment]
@mattleibow Yes, `IsSpellCheckEnabled` is false and `IsTextPredictionEnabled` is false.

**Comment by breps83** [comment]
I am having the same issue. No matter what I change in Maui I cannot get rid of the text suggestions on the keyboard.
